### PR TITLE
prices on linea

### PR DESCRIPTION
**Pull request by ppclunghe**
prices for Linea L2 (USDC, USDT, WETH, wstETH )

### CODE REVIEW

**Review by jeff-dude — COMMENTED** [review]
thanks for helping initiate this 🙏

File: models/prices/linea/prices_linea_tokens.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+
+    ('usdc-usd-coin', 'linea', 'USDC', 0x176211869cA2b568f2A7D4EE941E073a821EE1ff, 6),
+    ('usdt-tether', 'linea', 'USDT', 0xA219439258ca9da29E9Cc4cE5596924745e12B93, 6),
+    ('weth-weth', 'linea', 'WETH', 0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f, 18),
+    ('wsteth-wrapped-liquid-staked-ether-20', 'linea', 'wstETH', 0xB5beDd42000b71FddE22D3eE8a79Bd49A568fC8F, 18),
```
[the remark this answers is not in the document]
Comment (ppclunghe) [reply models/prices/linea/prices_linea_tokens.sql]: Fixed the comma, but I didn't find Linea in result of https://api.coinpaprika.com/v1/contracts endpoind, probably it is the issue

**Comment by jeff-dude** [comment]
fyi @viniabussafi @chef-seaweed 
prices spell for linea
i will look to help finalize and get in prod 🙏

File: models/prices/linea/prices_linea_tokens.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeff-dude) [reply models/prices/linea/prices_linea_tokens.sql]: yes, but we can workaround this. let me double check we're all good.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**